source/nodejs/adaptivecards/src/controls/adaptivecards-controls.css

```diff
@@ -0,0 +1,96 @@
+.ms-ctrl-overlay {
```

Comment:
I know this is going to be a bit painful, but could we change the `ms` prefix on all of those classes into `ac` since they are now part of the `adaptivecards` package?

Reply:
Actually, we need to make it even more painful, sorry :)

All these styles should be merged into the main AC CSS files. You will need to copy all those classes (after changing the prefix from ms to ac) into each of the CSS files that are under source\nodejs\adaptivecards-designer\src\containers\<all subfolders>\<all CSS files>